src/App.css

```diff
@@ -1,19 +1,105 @@
-.App {
-  text-align: center;
+/* App.css */
+.app-container {
+  display: flex;
+  flex-direction: column;
+  height: 100vh; /* height of 100vh ensures that the layout spans the full height */
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.task-list,
+.task-in-progress,
+.task-in-review,
+.task-completed {
+  flex: 1; /* Equal distribution of space */
+  box-sizing: border-box; /* Include padding and border in the width calculation */
+  width: calc(33.33% - 40px); /* 33.33% width with 20px margin on each side */
```

Comment:
this seems responsive but might not actually be. 40px on a mobile screen might be a lot.

Reply:
Will test it out